tests/ops/models/test_connectionconfig.py

```diff
@@ -146,3 +146,10 @@ def test_default_value_saas_config_invalid_type(
         saas_example_config["type"] = "invalid"
         with pytest.raises(ValueError):
             SaaSConfig(**saas_example_config)
+
+    def test_connection_type_human_readable(self):
+        with pytest.raises(ValueError):
+            ConnectionType("nonmapped_type").human_readable()
+
+        for connection in ConnectionType:
+            connection.human_readable  # Makes sure all ConnectionTypes have been added to human_readable mapping
```

Comment:
I would split these into 2 independent tests so you can run the tests once and get the results of both. Combined if the error test fails you have to run it again after fixing to make sure the connections are present. By splitting you can get the results of both in one run.

```suggestion
    def test_connection_type_human_readable(self):
        for connection in ConnectionType:
            connection.human_readable  # Makes sure all ConnectionTypes have been added to human_readable mapping
            
    def test_connection_type_human_readable_invalid(self):
        with pytest.raises(ValueError):
            ConnectionType("nonmapped_type").human_readable()
```